Author: Jonah Brüchert <jbb@kaidan.im>
--- plasma-settings-0.0.1+git20180430.orig/CMakeLists.txt
+++ plasma-settings-0.0.1+git20180430/CMakeLists.txt
@@ -32,7 +32,7 @@ set (QT_MIN_VERSION "5.4.0")
 
 find_package(KF5 REQUIRED COMPONENTS
     WindowSystem CoreAddons Solid Config Auth DBusAddons
-    I18n Plasma PlasmaQuick Declarative KIO KDELibs4Support
+    I18n Plasma PlasmaQuick Declarative KIO
     )
 
 find_package(KAccounts)
